pvm/ts/src/engine/actions/reRaiseAction.test.ts

```
@@ -0,0 +1,112 @@
+        bets.set(PLAYER_1, ONE_TOKEN); // 1 token (small blind)
+        bets.set(PLAYER_2, TWO_TOKENS); // 2 tokens (big blind)
+        bets.set(PLAYER_3, FIVE_TOKENS); // 5 tokens
+        bets.set(PLAYER_4, 1300000000000000000n); // 13 tokens
+        jest.spyOn(game, "getBets").mockReturnValue(bets);
+
+        // Mock addAction method on game
+        game.addAction = jest.fn();
+    });
+
+    describe("verify", () => {
+        it.only("should return correct range for a re raise", () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)